contracts/hackatom/src/msg.rs

```diff
@@ -66,20 +67,24 @@ pub enum ExecuteMsg {
     UserErrorsInApiCalls {},
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema, QueryResponses)]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 pub enum QueryMsg {
     /// returns a human-readable representation of the verifier
     /// use to ensure query path works in integration tests
+    #[returns(VerifierResponse)]
```

Comment:
this is amazing 👏🏻👏🏻👏🏻

very declarative! this is cool

Reply:
Love this.